BaseClasses.py

```diff
@@ -844,20 +844,29 @@ def add_locations(self, locations: Dict[str, Optional[int]], location_type: Opti
         for location, address in locations.items():
             self.locations.append(location_type(self.player, location, address, self))
 
-    def add_exits(self, exits: Dict[str, Optional[str]], rules: Dict[str, Callable[[CollectionState], bool]] = None) -> None:
+    def add_exits(self, exits: Union[Iterable[str], Dict[str, Optional[str]]],
+                  rules: Dict[str, Callable[[CollectionState], bool]] = None) -> None:
         """
         Connects current region to regions in exit dictionary. Passed region names must exist first.
 
         :param exits: exits from the region. format is {"connecting_region", "exit_name"}
         :param rules: rules for the exits from this region. format is {"connecting_region", rule}
         """
-        for exiting_region, name in exits.items():
-            ret = Entrance(self.player, name, self) if name \
-                else Entrance(self.player, f"{self.name} -> {exiting_region}", self)
-            if rules and exiting_region in rules:
-                ret.access_rule = rules[exiting_region]
-            self.exits.append(ret)
-            ret.connect(self.multiworld.get_region(exiting_region, self.player))
+        if isinstance(exits, Dict):
+            for exiting_region, name in exits.items():
+                ret = Entrance(self.player, name, self) if name \
+                    else Entrance(self.player, f"{self.name} -> {exiting_region}", self)
+                if rules and exiting_region in rules:
+                    ret.access_rule = rules[exiting_region]
+                self.exits.append(ret)
+                ret.connect(self.multiworld.get_region(exiting_region, self.player))
+        else:
+            for region in exits:
+                ret = Entrance(self.player, f"{self.name} -> {region}", self)
```

Comment:
I think for ER you'd pass entrance names in instead of relying on the automatically generated names, and this at least is clear what the connection is if it isn't explicitly named, which is the purpose.